### ISSUE TITLE
Pagination issue

### ISSUE THREAD

**Issue by cu12** [jantman/awslimitchecker#107]
Hi, I'm experiencing the following pagination issue:

 awslimitchecker -l
awslimitchecker 0.2.2 is AGPL-licensed free software; all users have a right to the full source code of this version. See <https://github.com/jantman/awslimitchecker>
WARNING:awslimitchecker.utils:Query returned a dict, but does not have _paginate_dict params set; cannot paginate (<bound method SupportConnection.describe_trusted_advisor_checks of SupportConnection:support.us-east-1.amazonaws.com>)
WARNING:awslimitchecker.utils:Query returned a dict, but does not have _paginate_dict params set; cannot paginate (<bound method SupportConnection.describe_trusted_advisor_check_result of SupportConnection:support.us-east-1.amazonaws.com>)
AutoScaling/Auto Scaling groups                        20 (API)
...

System details:
Collecting awslimitchecker
  Downloading awslimitchecker-0.2.2.tar.gz (87kB)
    100% |████████████████████████████████| 90kB 4.5MB/s
Collecting boto>=2.32.0 (from awslimitchecker)
  Downloading boto-2.38.0-py2.py3-none-any.whl (1.3MB)
    100% |████████████████████████████████| 1.3MB 276kB/s
Collecting termcolor>=1.1.0 (from awslimitchecker)
  Downloading termcolor-1.1.0.tar.gz
Collecting python-dateutil>=2.4.2 (from awslimitchecker)
  Downloading python_dateutil-2.4.2-py2.py3-none-any.whl (188kB)
    100% |████████████████████████████████| 192kB 851kB/s
Collecting six>=1.5 (from python-dateutil>=2.4.2->awslimitchecker)
  Downloading six-1.10.0-py2.py3-none-any.whl
Building wheels for collected packages: awslimitchecker, termcolor
  Running setup.py bdist_wheel for awslimitchecker
  Stored in directory: /Users/xxx/Library/Caches/pip/wheels/2a/63/15/08682535c442e9e0b896d5d0776711471f864e381a2134a74a
  Running setup.py bdist_wheel for termcolor
  Stored in directory: /Users/xxx/Library/Caches/pip/wheels/ad/7b/3a/b213d6375a91076f20681a1d97550d04b8a7a371f0316a79f1
Successfully built awslimitchecker termcolor
Installing collected packages: boto, termcolor, six, python-dateutil, awslimitchecker
Successfully installed awslimitchecker-0.2.2 boto-2.38.0 python-dateutil-2.4.2 six-1.10.0 termcolor-1.1.0

**Comment by jantman** [comment]
@cu12 sorry for the delayed response. Could you please run that command with `-vv` appended for debug-level output? The message that's being logged is rather strange, as there should be information about the query function in the ending `()`.

However, when I run against an account with Business-level support (and Trusted Advisor), I believe I can reproduce this:

```
(awslimitchecker)jantman@phoenix:pts/1:~/GIT/awslimitchecker (develop=)$ awslimitchecker -l
awslimitchecker 0.2.2@f5a0823 is AGPL-licensed free software; all users have a right to the full source code of this version. See <git@github.com:jantman/awslimitchecker.git>
WARNING:awslimitchecker.utils:Query returned a dict, but does not have _paginate_dict params set; cannot paginate (<bound method SupportConnection.describe_trusted_advisor_checks of SupportConnection:support.us-east-1.amazonaws.com>)
WARNING:awslimitchecker.utils:Query returned a dict, but does not have _paginate_dict params set; cannot paginate (<bound method SupportConnection.describe_trusted_advisor_check_result of SupportConnection:support.us-east-1.amazonaws.com>)
AutoScaling/Auto Scaling groups                        500 (API)
...
```

Assuming this is the same thing, this is a non-critical issue; Trusted Advisor doesn't paginate responses, the query will return all of the data that's needed by awslimitchecker. I'll assume your issue is the same as what I'm seeing above (though it's really confusing to me that the method representation is missing...). I'm going to be busy with family for the next few days, but I'll try to get a fix out tonight.

**Comment by jantman** [comment]
@cu12 I believe I have a fix for this ready; if you have time, I'd like to see if this fixes your issue. If you can test it, I'll cut a release today or tomorrow. If not, it will wait for the next normal release.

Assuming you installed inside a virtualenv, can you please source that venv and then:

```
pip uninstall -y awslimitchecker
pip install -e git+https://github.com/jantman/awslimitchecker.git@issues/107#egg=awslimitchecker
```

Thanks,
Jason

**Comment by cu12** [comment]
@jantman This seems to be solving the pagination problem above, thanks for the quick fix.

**Comment by jantman** [comment]
@cu12 this has been released as 0.2.3 and is now live on pypi. Thanks for bringing this to my attention!
